integr8ly/integreatly-operator
# add solution explorer 0.0.52 manifest

- Include Solution Explorer manifest for 0.0.52
- Remove manifest for 0.0.47
- Ensure 0.0.52 manifest "replaces" 0.0.48

verification:
- ensure installation result sin 0.0.52 of the solution explorer
- ensure upgrade from rhmi 2.1.0 works as expected
